library/defrag.c

```diff
@@ -193,10 +193,11 @@ int eblob_defrag(struct eblob_backend *b)
 			continue;
 		}
 
-		/* skips sorted bases if defrag for them is not needed. Always defrag unsorted bases and
+		/* skips sorted bases if defrag for them is not needed. Defrag unsorted bases and
```

Comment:
This should include description of both behaviour with `EBLOB_DEFRAG_STATE_DATA_SORT` and with  `EBLOB_DEFRAG_STATE_DATA_COMPACT`